README.md

```diff
@@ -79,76 +90,118 @@ dependencies and compiling is simple.  Just run:
+    for (instance <- instances) {
+      s.append("Instance: " + instance.toString() + "\n")
+    }
+    println(s.toString())
 
-`openie` takes a number of command line arguments.  To see them all run
-`java -jar openie-assembly.jar --usage`.  Of particular interest are
-`--ignore-errors` which continues running even if an exception is encountered, `--binary` which gives the binary(triples) output and `--split` which splits the input document text into sentences.
+That wIll produce this output:
 
-There are two formats--a simple format made for ease of reading and a
-columnated format used for machine processing.  The format can be specified
-with either `--format simple` or `--format column`.  The simple format is
-chosen by default.
+    Instance: 0.93 (U.S. President Obama; gave; a speech)
+    Instance: 0.88 (Obama; [is] President [of]; United States)
+
+#### SimpleFormat
+
+OpenIECli provides a SimpleFormat helper that produces similar output:
+
+    val sentence = "U.S. President Obama gave a speech"
+    val instances = openie.extract(sentence)
+    val sw = new StringWriter()
+    SimpleFormat.print(new PrintWriter(sw), sentence, instances)
+    println(sw.toString())
+
+That will produce this output:
 
-### Java Demo
+    U.S. President Obama gave a speech
+    0.93 (U.S. President Obama; gave; a speech)
+    0.88 (Obama; [is] President [of]; United States)
 
-A simple java demo which uses openIE (https://github.com/OpenIE-HelperCodes/OpenIEDemo1)
+#### ColumnFormat
 
-#### Simple Format
+OpenIECli also provides a ColumnFormat helper that produces output with tabs:
 
-```
-> John ran down the road to fetch a pail of water.
-John ran down the road to fetch a pail of water.
-0.86 (John; ran; down the road; to fetch a pail of water)
-0.82 John ran:(John; ran down the road to fetch; a pail of water)
-```
+    val sentence = "U.S. President Obama gave a speech"
+    val instances = openie.extract(sentence)
+    val sw = new StringWriter()
+    ColumnFormat.print(new PrintWriter(sw), sentence, instances)
+    println(sw.toString())
 
-#### Columnated Format
+That will produce this output:
 
-Columns are separated by tab, making it hard to read in this README.
+    0.9329286852051247              SimpleArgument(U.S. President Obama,List([0, 20)))      Relation(gave,List([21, 25)))   SimpleArgument(a speech,List([26, 34))) U.S. President Obama gave a speech
+    0.8847999636040884              SimpleArgument(Obama,List([15, 20)))    Relation([is] President [of],List([5, 14)))     SimpleArgument(United States,List([0, 4)))      U.S. President Obama gave a speech
 
-```
-0.8576784836790008	John	ran	down the road; to fetch a pail of water	John ran down the road to fetch a pail of water.
-0.8195727266148489	John ran	John	ran down the road to fetch	a pail of water	John ran down the road to fetch a pail of water.
-```
+## Usage at AllenAI
+
+Built jars of `openie-standalone` are published to AllenAI's private repository
```

Comment:
Any reason to publish to private BinTray? the repo is public so anyone interested has direct access to the code of course, but is there a reason not to publish the library publicly?